app/controllers/application_controller.rb

```diff
@@ -150,9 +151,10 @@ def post_or_reply_link(reply)
 
   def posts_from_relation(relation, no_tests: true, with_pagination: true, select: '', max: false)
     if max
-      posts = relation.select('posts.*, max(boards.name) as board_name, max(users.username) as last_user_name, bool_or(users.deleted) as last_user_deleted'+ select)
+      posts = relation.select("posts.*, max(boards.name) as board_name, max(users.username) as last_user_name, \
```

Comment:
Can we concatenate strings here, rather than escaping the newline? (I think this is going to add whitespace to the SQL query, if we ever look at it, and I find newline escaping kinda strange)